benchmark/kvbench/README.md

```
@@ -343,6 +404,31 @@ Example matrix file:
+0 710M 0 0
+0 0 0 0
+
+[read]
+710M 710M 0 0
+
+[write]
+710M 710M 0 0
+```
+
+Reference it from YAML with `tp_file`:
+```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)